taxi_zebra/commands.py

```diff
@@ -76,4 +79,5 @@ def balance(ctx):
     click.echo("Hours balance: {}".format(signed_number(hours_balance)))
     click.echo("Hours balance after push: {}".format(signed_number(hours_balance + hours_to_be_pushed)))
     click.echo("Hours done this week: {:.2f}".format(total_duration))
+    click.echo("Hours done today: {:.2f}".format(total_duration_today + hours_to_be_pushed))
```

Comment:
Why add the hours to be pushed? In the case where you have a pending timesheet from yesterday, you'd get yesterday's hours + today's, no?

Reply:
Yeah you're right, I didn't think of that. I separated the hours to be pushed from the output in the next commit.